lms/static/scripts/frontend_apps/utils/prop-types.js

```diff
@@ -0,0 +1,75 @@
+import { isValidElement } from 'preact';
```

Comment:
I'm wondering if this file be in frontend-toolkit repo?

Reply:
Yes, I think it would make sense to move it to that repo in future. I'm putting it here to start with to validate it in the context of a concrete use case.

Reply:
roger that